backend/logic/courses/courses.js

```diff
@@ -261,12 +263,30 @@ var updateCourse = async (course, newTitle, newTags) => {
   }
 };
 
+let phaseOutCourse = async courseCode => {
+  let connection = await mysql.getNewConnection();
+  try {
+    await format.verifyCourse(courseCode);
```

Comment:
This can be moved outside of the try block since you're rethrowing the error anyways